modules/installation-rhv-downloading-ansible-playbooks.adoc

```diff
@@ -25,35 +24,22 @@ $ cd playbooks
 +
 [source,terminal,subs=attributes+]
 ----
-$ curl -s -L -X GET https://api.github.com/repos/openshift/installer/contents/upi/ovirt?ref=release-<version>  |
-grep 'download_url.*\.yml' |
-awk '{ print $2 }' | sed -r 's/("|",)//g' |
-xargs -n 1 curl -O
+$  xargs -n 1 curl -O <<< '
```

Comment:
Just checking - this has an opening `'`, just wondering if there was supposed to be a closing `'` after the list of files

Reply:
Oops! thanks for catching that!